.github/.env

```diff
@@ -5,3 +5,4 @@ AZURE_RESOURCE_GROUP_LOCATION=australiaeast
 BICEP_PATH=./infra/main.bicep
 IMAGE_NAME=sswwebsite
 KEY_VAULT=kv-sswwebsite-9eb3
+NEXT_PUBLIC_APP_INSIGHT_INSTRUMENTATION_KEY=9fce8146-edbc-46d4-8542-74da9fb75503
```

Comment:
instrumentation keys have been deprecated - we should be using connection strings
I think it would make more sense to grab this value at build time as opposed to committing it - it is more likely to change than an resource name